content/docs/atom-guide/features/taa.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Temporal Anti-Aliasing
```

Comment:
```suggestion
title: Temporal Anti-aliasing
```
When capitalizing "anti-aliasing", even in a title, only "Anti-" is capitalized.